app/services/permission_service.rb

```diff
@@ -43,6 +44,7 @@ def store_admin_permissions(controller, action)
       return true if controller == 'stores/dashboard' && action == 'show'
       return true if controller == 'order_items'      && action == 'update'
       return true if controller == 'stores'           && action.in?(%w(index show edit update))
+      return true if controller == 'orders'           && action == 'show'
```

Comment:
Operator `&&` should be surrounded with a single space.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.